src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+
+    private StringBuilder stringBuilder = new StringBuilder();
```

Comment:
shouldn't be class level field, may impact issues
```suggestion
    private StringBuilder stringBuilder = new StringBuilder();
```